src/main/resources/assets/jabba/lang/ru_RU.lang

```diff
@@ -79,6 +79,8 @@ text.jabba.waila.key.content=Содержимое бочки
 text.jabba.waila.key.quantity=Количество предметов
 text.jabba.waila.key.stacks=Макс. стаков
 
-text.jabba.tooltip.slots.used=Использовано слотов : 
-text.jabba.tooltip.slots.provided=Слотов предоставлено : 
-text.jabba.tooltip.empty=< Пустая >
\ No newline at end of file
+text.jabba.tooltip.slots.used=Использовано слотов :
+text.jabba.tooltip.slots.provided=Слотов предоставлено :
+text.jabba.tooltip.empty=< Пустая >
+
+mcp.mobius.betterbarrel=Barrel
```

Comment:
You don't need to add this to each lang file, if it is missing, MC will use the en_US localization

Reply:
yeah but if it's missing and someone wants to translate it, he has to do a diff with the en_US localization. Making it explicitely not translated is more convenient imo and doesn't change anything.

Reply:
Yeah okay that is true